React/CoreModules/RCTAlertController.m

```diff
@@ -20,7 +20,18 @@ @implementation RCTAlertController
 - (UIWindow *)alertWindow
 {
   if (_alertWindow == nil) {
-    _alertWindow = [[UIWindow alloc] initWithFrame:RCTSharedApplication().keyWindow.bounds];
+    if (@available(iOS 13.0, *)) {
+        for (UIWindowScene *scene in RCTSharedApplication().connectedScenes) {
```

Comment:
For app which supports CarPlay, `RCTSharedApplication().connectedScenes` will contain the `CPTemplateApplicationScene` instance.

So maybe here scene should be `UIScene *`, then we can cast it to `UIWindowScene *` after type checking?